spec.html

```diff
@@ -41429,7 +41426,7 @@ <h1>The Strict Mode of ECMAScript</h1>
       `implements`, `interface`, `let`, `package`, `private`, `protected`, `public`, `static`, and `yield` are reserved words within strict mode code. (<emu-xref href="#sec-reserved-words"></emu-xref>).
     </li>
     <li>
-      A conforming implementation, when processing strict mode code, must not extend, as described in <emu-xref href="#sec-additional-syntax-numeric-literals"></emu-xref>, the syntax of |NumericLiteral| to include |LegacyOctalIntegerLiteral|, nor extend the syntax of |DecimalIntegerLiteral| to include |NonOctalDecimalIntegerLiteral|.
+      A conforming implementation, when processing strict mode code, must disallow instances of the productions <emu-grammar>NumericLiteral :: LegacyOctalIntegerLiteral</emu-grammar> and <emu-grammar>DecimalIntegerLiteral :: NonOctalDecimalIntegerLiteral</emu-grammar>.
```

Comment:
is this paragraph necessary given the syntax errors described above?

Reply:
Is *anything* in Annex C necessary? I thought it just summarized restrictions stated normatively elsewhere.

Reply:
lol fair enough.